tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3134,4 +3134,27 @@ public function publicDownloadsTheFolderFromTheLastCreatedPublicLink(string $res
 			)
 		);
 	}
+
+	/**
+	 * @Then the relative quota amount should be :quota_amount
+	 *
+	 * @param string $quotaAmount
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theRelativeQuotaAmountShouldBe($quotaAmount) {
+		$data = $this->ocsContext->getOCSResponseData($this->featureContext->getResponse());
+		if (isset($data->quota, $data->quota->relative)) {
```

Comment:
```suggestion
		if (\isset($data->quota, $data->quota->relative)) {
```

Reply:
Could not find \isset used anywhere else.

Reply:
yeah, we should have used that. it separates the namespace scope and prevents the functions clash (may not happen very often though)